projects/prison-custody-status-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/release/PersonDied.kt

```diff
@@ -28,8 +28,8 @@ class PersonDied(
         val people = personRepository.findByNomsNumberAndSoftDeletedIsFalse(nomsId)
         val person = when (people.size) {
             1 -> people[0]
-            0 -> throw NotFoundException("Person", "nomsNumber", nomsId)
-            else -> throw IllegalStateException("More than one case with the same Noms Number: $nomsId")
+            0 -> throw IgnorableMessageException("MissingNomsNumber: $nomsId")
+            else -> throw IgnorableMessageException("DuplicateNomsNumber: $nomsId")
```

Comment:
I think the exception message is what gets added to app insights, so appending the $nomsId will make it harder to aggregate the events.  The noms number is added as a Telemetry property anyway so we can still see the value if we need to.